src/ctapipe/irf/spectra.py

```diff
@@ -0,0 +1,26 @@
+"""Definition of spectra to be used to calculate event weights for irf computation"""
+
+from enum import Enum
+
+import astropy.units as u
+from pyirf.spectral import CRAB_HEGRA, IRFDOC_ELECTRON_SPECTRUM, IRFDOC_PROTON_SPECTRUM
```

Comment:
wasn't this moved to a new package? Or is that just handled inside pyirf?

Reply:
I don't see any recent change regarding this in pyirf. Am I missing something?

Reply:
maybe it's just a proposal. I seem to remember somebody making a common package for the spectra, but I can't recall who or where it is! (maybe @maxnoe  remembers), but in any case, it's not a problem here if it's still in PyIRF

Reply:
Is it https://gitlab.cta-observatory.org/cta-computing/dpps/cosmic-ray-spectra?